storage/simulation/cluster.go

```diff
@@ -134,13 +216,12 @@ func (c *Cluster) String() string {
 		}
 	}
 
-	var nodeIDs []int
+	var nodeIDs []proto.NodeID
 	for nodeID := range c.nodes {
-		nodeIDs = append(nodeIDs, int(nodeID))
+		nodeIDs = append(nodeIDs, nodeID)
 	}
-	sort.Ints(nodeIDs)
+	sort.Sort(proto.NodeIDSlice(nodeIDs))
```

Comment:
When looking for examples, this seemed to be the most common way to add sorts to slices.  I didn't see any instances of declaring the variable to be the type with the interface.  
I could declare it as NodeIDSlice, but I like that it's just an []proto.NodeID.  That seems clearer to me.  
If you feel strongly, I can change it up.